### DISCUSSION TITLE
Disable cores as additional powersaving technique

### DISCUSSION THREAD

**Discussion by richardbowman** [AdnanHodzic/auto-cpufreq#493] [General]
Hi, I'm wondering if you had considered (or if it is already) shutting off CPU cores entirely. I've been using another tool CPUFreq Manager and it lets me set the max enabled cores while in power saving modes. It seems to have a substantial effect on battery life.

**Comment by AdnanHodzic** [comment]
There was a [feature request](https://github.com/AdnanHodzic/auto-cpufreq/issues) for this functionality, but unfortunately now I can't find it. Hence, please feel free to create it and I hope someone will take a look at it eventually :slightly_smiling_face: 

If you want to give it a try in implementing this functionality, please give it a try and [contribute to the project](https://github.com/AdnanHodzic/auto-cpufreq/#code-contribution) and you will be credited for your work as part of [future release](https://github.com/AdnanHodzic/auto-cpufreq/releases).
